dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -840,6 +891,19 @@ private boolean canDeleteWorkflow(String path) {
         }
     }
 
+    protected List<Token> checkOnBitbucketToken(User user) {
```

Comment:
This looks like this was it copied from AbstractWorkflowResource. We should avoid the duplication, and rename it in the process -- when I saw its invocation above, I started to enter a comment, based on the name, that it was only going return BitBucket tokens.

Not for this PR, but for down the road, if we get more BitBucket users, we  should consider updating this method, which is why it would be good if it only existed once -- we shouldn't be generating a new access token every time, but only when the curent one has  expired or is close to expired.